src/scss/_elements.root.scss

```diff
@@ -8,3 +8,12 @@
   
   --size: 10rem;
 }
+
+::selection {
```

Comment:
Never seen something like this. What's it doing?

Reply:
Oh, it’s the way you can style the text selection colors when text is highlighted by the cursor.

Reply:
Nice! Looks awesome.